CONTRIBUTING.md

```diff
@@ -18,8 +18,10 @@ is much appreciated in any of the following topics:
 Adding a new function will be a huge help for the library growth and
 should not be too problematic for the first issue. Extending the
 library of translations is also a good task to start with.
+[Here](https://docs.google.com/spreadsheets/d/1UUskn4ZDDjLGSpO6kg73DOvabNoeqLbkJYyVfLyYlYw)
+you can find a list of functions' translations.
 
-Visit [building](building.md) and [testing](testing.md) sections to
+Visit [building](https://handsontable.github.io/hyperformula/guide/building.htm) and [testing](https://handsontable.github.io/hyperformula/guide/testing.html) sections to
```

Comment:
Awesome! But we should use a redirect, so the links will stay correct after we move the docs to our server
```suggestion
Visit [building]( https://handsontable.com/docs/hyperformula/guide/building.htm) and [testing]( https://handsontable.com/docs/hyperformula/guide/testing.html) sections to
```